Nightly TF 1.14.1.dev20190607 breaks v0.16.3

**Environment:**
1. Framework: TensorFlow 
2. Framework version: 1.14.1.dev20190607
3. Horovod version: 0.16.3
4. MPI version: 2.0.2
5. CUDA version: 10.0
6. NCCL version: 2.4.2
7. Python version: 3.5m
8. OS and version: Debian 9.8

**Bug report:**

Build Horovod with latest TF nightly fails with the following error:

```
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/build/python3.5-3.5.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DEIGEN_MPL2_ONLY=1 -Ithird_party/eigen -Ithird_party/lbfgs/include -Ithird_party/boost/assert/include -Ithird_party/boost/config/include -Ithird_party/boost/core/include -Ithird_party/boost/detail/include -Ithird_party/boost/iterator/include -Ithird_party/boost/lockfree/include -Ithird_party/boost/mpl/include -Ithird_party/boost/parameter/include -Ithird_party/boost/predef/include -Ithird_party/boost/preprocessor/include -Ithird_party/boost/static_assert/include -Ithird_party/boost/type_traits/include -Ithird_party/boost/utility/include -Ithird_party/flatbuffers/include -I/usr/include/python3.5m -I/home/byronyi/.virtualenv/tf/include/python3.5m -c horovod/tensorflow/mpi_ops.cc -o build/temp.linux-x86_64-3.5/horovod/tensorflow/mpi_ops.o -std=c++11 -fPIC -O2 -Wall -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent/include -I/usr/lib/x86_64-linux-gnu/openmpi/include -pthread -L/usr//lib -L/usr/lib/x86_64-linux-gnu/openmpi/lib -lmpi_cxx -lmpi -I/home/byronyi/.virtualenv/tf/lib/python3.5/site-packages/tensorflow/include -D_GLIBCXX_USE_CXX11_ABI=0
  cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
  horovod/tensorflow/mpi_ops.cc:22:42: fatal error: tensorflow/core/framework/op.h: No such file or directory
   #include "tensorflow/core/framework/op.h"
                                            ^
  compilation terminated.
  INFO: Unable to build TensorFlow plugin, will skip it.
```

--- a/tensorflow/tools/pip_package/build_pip_package.sh
+++ b/tensorflow/tools/pip_package/build_pip_package.sh
@@ -176,6 +176,18 @@ function build_wheel() {
   rm -f MANIFEST
   echo $(date) : "=== Building wheel"
   "${PYTHON_BIN_PATH:-python}" setup.py bdist_wheel ${PKG_NAME_FLAG} >/dev/null
+
+  # Fix include path by moving tensorflow_core/include/tensorflow{_core,}
+  WHEEL_TMPDIR=$(mktemp -d)
+  WHEEL=$(readlink -f dist/*.whl)
+  unzip $WHEEL -d $WHEEL_TMPDIR > /dev/null
+  rm $WHEEL
+  pushd $WHEEL_TMPDIR > /dev/null
+  mv tensorflow_core/include/tensorflow{_core,}
+  zip -r $WHEEL . > /dev/null
+  popd > /dev/null
+  rm -rf $WHEEL_TMPDIR
+
   mkdir -p ${DEST}
   cp dist/* ${DEST}
   popd > /dev/null

--- a/tensorflow/tools/pip_package/setup.py
+++ b/tensorflow/tools/pip_package/setup.py
@@ -141,7 +141,7 @@ class InstallCommand(InstallCommandBase):
 
   def finalize_options(self):
     ret = InstallCommandBase.finalize_options(self)
-    self.install_headers = os.path.join(self.install_purelib, 'tensorflow_core',
+    self.install_headers = os.path.join(self.install_platlib, 'tensorflow_core',
                                         'include')
     self.install_lib = self.install_platlib
     return ret

